akka-docs/rst/common/circuitbreaker.rst

```diff
@@ -53,50 +53,56 @@ What do they do?
 
 .. image:: ../images/circuit-breaker-states.png
 
-========
+
+===============
 Examples
-========
+===============
 
 --------------
 Initialization
 --------------
 
-Here's how a :class:`CircuitBreaker` would be configured for:
+Following are examples for initializing a :class:`CircuitBreaker` pattern with the following configuration:
   * 5 maximum failures
   * a call timeout of 10 seconds 
   * a reset timeout of 1 minute
 
-^^^^^^^
+.. note::
+	
+	There is also a :class:`CircuitBreakerProxy` actor that you can use, it is an alternative implementation of the pattern, where the main difference is that it is intended to be used only for request-reply interactions with another actor. See :ref:`Circuit Breaker Actor <circuit-breaker-proxy>`
```

Comment:
This is not the most important thing to note, I'd move it down after the samples, but before the low level api section.

minor: I think we usually line break these before 120 chars